python/test/test_meas.py

```diff
@@ -221,3 +221,21 @@ def test_alp_histogram():
     print(alp_qbl("B"))
     print(alp_qbl("C"))
     print(alp_meas.map(1))
+
+def test_randomized_response_bitvec():
+    np = pytest.importorskip('numpy')
+
+    m_rr = dp.m.make_randomized_response_bitvec(
+        dp.bitvector_domain(max_weight=4), dp.discrete_distance(), f=0.95
+    )
+
+    data = np.packbits(
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0]
+    )
+    assert np.array_equal(data, np.array([0, 8, 12], dtype=np.uint8))
```

Comment:
Checking my comprehension, you could add more zero-bits to the end without changing `data`?

Reply:
Yes, the input is always padded out with zeros.